docs/acp/user/intro-to-acp/syncapi.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SyncAPI
+Descriptions: 
+template: howto-guide-template
+---
+
+Spryker uses schema files to generate code for your project including predefined test cases. The purpose of doing so is to let you focus on building your business logic without caring about the boilerplate code.
+
+## Code generation
+
+You can use the [SyncAPI’s](https://github.com/spryker-sdk/sync-api/) OpenAPI schema file to generate code with the help of a [Spryk](/docs/sdk/dev/spryks/spryks.html).
```

Comment:
@stereomon what is actually the SyncAPI?

Reply:
That's about the synchronous API that we support e.g. The JSON API.

Reply:
also known as Glue API in Spryker :)